src/main/java/test/gc/switchele/commands/SwitchElement.java

```diff
@@ -53,11 +53,7 @@ private boolean changeAvatarElement(Player sender, int avatarId, Element element
     @Override
     public void execute(Player sender, Player targetPlayer, List<String> args) {
         if (args.size() != 1) {
-            CommandHandler.sendMessage(sender, "Usage: /se OR /switchelement [White/Anemo/Geo/Electro/Dendro]");
-            return;
-        }
-        if (sender == null) {
-            Switchele.getInstance().getLogger().info("SwitchElement command couldn't be called by console.");
+            sendUsageMessage(sender);
```

Comment:
sendUsageMessage is not in 1.2.2-dev and because most 3.0 clients are working on 1.2.2-dev I don't want to make this update (yet)